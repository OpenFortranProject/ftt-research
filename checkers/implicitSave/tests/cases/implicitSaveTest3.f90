subroutine h
  integer :: x = 4
  print *,x
end subroutine

program main
  print *, "Testing h"
  call h
  call h
  call h
end program main
