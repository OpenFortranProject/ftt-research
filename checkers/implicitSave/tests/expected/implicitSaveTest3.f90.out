Warning: environment variable COMPASS_PARAMETERS is not set, trying to use compass_parameters of the current path
 Running Prerequisite SgProject
